metadata.pl

```diff
@@ -0,0 +1,273 @@
+# not be used or stored in scenario.json
+my @deprecated_properties = qw( forecast.ensemblesize asgs.path.fromdir 
+   asgs.path.lastsubdir asgs.enstorm enstorm hostname instance pseudostorm 
+   intendedAudience asgs.path.advisdir asgs.path.stormdir path.advisdir
+   path.stormdir config.forcing.nam.schedule.forecast.forecastcycle
+   config.forcing.nam.backsite config.forcing.nam.backdir config.forcing.nam.forecastlength
+   config.forcing.nam.reprojection.ptfile config.forcing.nam.local.altnamdir storm
+   stormnumber cpurequest ncpu numwriters h0 asgs prodID sea_surface_height_above_geoid
+   wind year ADCIRCGrid ColdStartTime Model RunEndTime RunStartTime RunType WindModel
+   currentcycle currentdate gusts
+);
+# the following properties from run.properties are array valued and designated in 
+# run.properties file with ( ) and should be stored as an array in json
+# e.g.:
+# post.executable.postprocess : (  createMaxCSV.sh includeWind10m.sh createOPeNDAPFileList.sh opendap_post.sh )
+my @paren_properties = qw ( post.executable.postprocess post.opendap.tds 
+   hpc.job.padcswan.jobenv hpc.job.prep15.jobenv hpc.job.prep15.subshellpids
+   hpc.job.prep15.nodelist hpc.job.padcswan.jobenv hpc.job.padcswan.subshellpids 
+   hpc.job.padcswan.nodelist post.opendap.files post.opendap.lsu_tds.linkablehosts
+   post.opendap.lsu_tds.copyablehosts
+);
+# the following properties from run.properties are array valued and designated
+# in run.properties as comma separated and should be stored as an array in json
+my @comma_properties = qw( notification.email.activate_list
+   notification.email.new_advisory_list notification.email.post_init_list
+   notification.email.job_failed_list notification.hpc.email.notifyuser
+   notification.opendap.email.opendapnotify 
+);
+# the following properties from run.properties are stored as individual
+# top level properties but should be considered a sublist of related 
+# metadata with a list of properties associated with them in turn
+# (to start off with, " File Name" and " Format")
+# adcirc.file.output.list
+my @outputfile_properties = ( 
+    "Water Surface Elevation Stations", "Water Surface Elevation",
+    "Water Current Velocity", "Barometric Pressure Stations",
+    "Wind Velocity Stations", "Barometric Pressure", "Wind Velocity",
+    "Maximum Water Surface Elevation", "Maximum Current Speed",
+    "Maximum Wind Speed", "Minimum Barometric Pressure",
+    "Maximum Wave Radiation Stress", "Mean Wave Direction",
+    "Maximum Mean Wave Direction", "Significant Wave Height",
+    "Maximum Significant Wave Height", "Mean Wave Period",
+    "Maximum Mean Wave Period", "Peak Wave Period",
+    "Maximum Peak Wave Period", "Initially Dry", "Inundation Time",
+    "Maximum Inundation Depth", "Ever Dried", "End Rising Inundation",
+    "Wind Velocity 10m Stations", "Wind Velocity 10m",
+    "Maximum Wind Speed 10m",
+);
+
+our $this = "metadata.pl"; 
+#
+GetOptions(
```

Comment:
(FYI) - I've been using `GetOptions` like this more:

```
my %opts = ();
my @flags = (qw/
  metadatafile=s
  keys=s
  mapscalar=s
  jsonify
/);
my $ok = GetOptions(\%opts, @flags);
```
Take a look at `cloud/general/asgs-brew.pl` around line 120 for a real example.